config/zones.json

```diff
@@ -800,70 +800,38 @@
     },
     "timezone": null
   },
-  "CL-SIC": {
+  "CL-SING-SIC": {
```

Comment:
from a quick Google search it seems like this is now merged into a zone called "SING". I might be wrong but I think SIC doesn't exist anymore.